LAUNCH Test Project/Assets/Scripts/CameraPos.cs

```diff
@@ -0,0 +1,47 @@
+﻿using UnityEngine;
+using UnityEngine.UI;
+
+namespace CameraToolkit
+{
+	public class CameraPos : MonoBehaviour
+	{
+        public Interpolator optionalInterpolator;
```

Comment:
Add some tooltips to your fields here, take a look here at the documentation for [Tooltip] https://docs.unity3d.com/ScriptReference/TooltipAttribute.html